charts/chronograf/README.md

```diff
@@ -1,58 +1,52 @@
-# Chronograf
+# Chronograf Helm chart
 
-##  An Open-Source Time Series Visualization Tool
+[Chronograf](https://github.com/influxdata/chronograf) is an open-source web application used to visualize your monitoring data and easily create alerting and automation rules.
 
-[Chronograf](https://github.com/influxdata/chronograf) is an open-source web application built by the folks over at [InfluxData](https://influxdata.com) and written in Go and React.js that provides the tools to visualize your monitoring data and easily create alerting and automation rules.
-
-## QuickStart
-
-```bash
-helm repo add influxdata https://helm.influxdata.com/
-helm upgrade --install chronograf influxdata/chronograf --namespace monitoring
-```
-
-## Introduction
-
-This chart bootstraps a Chronograf deployment and service on a Kubernetes cluster using the Helm Package manager.
+The Chronograf Helm chart uses the [Helm](https://helm.sh) package manager to bootstrap a Chronograf deployment and service on a [Kubernetes](http://kubernetes.io) cluster.
 
 ## Prerequisites
 
 - Kubernetes 1.4+
-- PV provisioner support in the underlying infrastructure (optional)
+- (Optional) PersistentVolume (PV) provisioner support in the underlying infrastructure
 
-## Installing the Chart
+## Install the chart
 
-To install the chart with the release name `my-release`:
+1. Add the InfluxData Helm repository:
 
-```bash
-helm upgrade --install my-release influxdata/chronograf
-```
+   ```bash
+   helm repo add influxdata https://helm.influxdata.com/
+   ```
+
+2. Run the following command, providing a name for your Chronograf release:
 
-The command deploys Chronograf on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+   ```bash
+   helm upgrade --install my-release influxdata/chronograf
+   ```
+    
+   > **Tip**: `--install` can be shortened to `-i`.
 
-> **Tip**: List all releases using `helm list`
+   This command deploys Chronograf on the Kubernetes cluster using the default configuration. To find parameters you can configure during installation, see [Configure the chart](#configure-the-chart).
 
-## Uninstalling the Chart
+  > **Tip**: To view all Helm chart releases, run `helm list`.
 
-To uninstall/delete the `my-release` deployment:
+## Uninstall the chart
+
+To uninstall the `my-release` deployment, use the following command:
 
 ```bash
 helm uninstall my-release
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
-
-## Configuration
+This command removes all Kubernetes components associated with the chart and deletes the release.
 
-The configurable parameters of the Chronograf chart and
-their descriptions can be seen in `values.yaml`. The [full image documentation](https://quay.io/influxdb/chronograf) contains more information about running Chronograf in docker.
+## Configure the chart
```

Comment:
This Chronograf Helm chart appears to have a few discrepancies between values.yaml and readme. A couple items of note:

- In readme, default value for image.repository is “quay.io/influxdb/chronograf”. This parameter doesn’t exist in the `values.yaml` file. Do we know why?
- Should accessModes be singular or plural? Should its default value [ReadWriteOnce] include brackets?
- Service.replicas isn’t included in the readme—intentional?
- Are `resources` parameters non-configurable?
- In value.yaml, persistence.enabled is false by default, readme says true.

Reply:
Thoughts on removing config parameters from README files? ...so we don’t have to maintain in two places

Reply:
1. It should be in values.yaml, commented out (As it's the default and not mandatory)
2. Plural and yes, `[]` signifies a list and `ReadWriteOnce` is the only element
3. It should be
4. Resources are configurable, there should be `limits` and `requests`, each with keys for `memory` and `cpu`: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits
5. If it's false in the `values.yaml`, lets go with that as default and update readme

Reply:
- Added default image.repository to `values.yaml`
- Updated accessMode to plural `values.yaml` 
- @rawkode, should we add brackets to ReadWriteOnce in `values.yaml`...or is having as a list optional and/or not default?
- Added service.replicas to README
- @rawkode --for service.replicas, is this description accurate: "number of replicas for the specified service.type"?
- added resources parameters to README
- updated persistence.enabled is false in README